util/ddlog_initdb

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env python
+# Generate create table statement given a ddlog exported schema and a table name.
+# Usage: ddlog_initdb SCHEMA.JSON TABLE_NAME
```

Comment:
Let's give a better name to this script.  How about `schema_json_to_sql`?

I was originally thinking this SQL generator should go under each driver, e.g.,  to handle `DISTRIBUTED BY` in GP and so on.  If you agree, I think it'll be a matter of just moving this to `shell/driver.postgresql/` and keeping a symlink or clone under `driver.mysql/`.